tools/redirect_cc/redirect_cc.cc

```diff
@@ -163,6 +167,16 @@ class RedirectCC {
     auto process = base::LaunchProcess(to_launch, options);
     int exit_code = -1;
     process.WaitForExit(&exit_code);
+#if BUILDFLAG(IS_WIN)
+    // To check the redirected file timestamp, it should be marked as dependency
+    // for ninja. Linux/MacOS gcc deps format includes this file properly.
+    // Windows msvc deps format does not include it, so we do it manually here.
+    if (exit_code == 0 && !brave_path.empty()) {
+      // This is a specially crafted string that ninja will look for to create
+      // deps.
+      std::wcerr << L"Note: including file: " << brave_path.c_str() << L"\n";
```

Comment:
basically reimplement this change: https://github.com/brave/brave-core/commit/becdcde3537d8958c41f87a428ffa59c1b9f1177